db/seeds.rb

```diff
@@ -0,0 +1,19 @@
+module Shop
```

Comment:
seed is good idea if you will keep here json or hash or some other data structure, and then load it in some other place